packages/atomic/src/components/facets-v1/atomic-rating-facet/atomic-rating-facet.tsx

```diff
@@ -115,8 +122,25 @@ export class AtomicRatingFacet
       .length;
   }
 
+  private generateCurrentValues() {
+    const scaleFactor = this.maxValue / this.numberOfStars;
+    const currentValues: NumericRangeRequest[] = [];
+    for (let i = 0; i < this.numberOfStars; i++) {
+      currentValues.push(
+        buildNumericRange({
+          start: i * scaleFactor,
```

Comment:
Should we round the number to e.g. two decimals, to avoid recurring decimals like `1.33333...`?

Reply:
Yes, I think that would be neater